### DISCUSSION TITLE
How do I remap different localization letters to their positional equivelents in latin QWERTY

### DISCUSSION THREAD

**Discussion by xxfogs** [NvChad/NvChad#2777] [Q&A]
I often work with Russian language and I thought that it would be really handy if I could map Russian letters to their latin QWERTY positional equivalents, largely because I often forget to change the localization.

So what exactly do I mean by that? Here's an example of QWERTY keyboard with Russian and English letters

![image](https://github.com/NvChad/NvChad/assets/103338067/667ecdf1-57b6-42c6-a9e6-b6c8f2cd8c56)

I want it so when I click, "н" it converts into "y" (only in normal mode). That's quite simple and I managed to get that done with the following code

<details>

<summary>Uncollapse me to see the code</summary>

```lua
-- Add Russian to Latin keybindings for all letters
local russian_to_latin = {
  ["й"] = "q",
  ["ц"] = "w",
  ["у"] = "e",
  ["к"] = "r",
  ["е"] = "t",
  ["н"] = "y",
  ["г"] = "u",
  ["ш"] = "i",
  ["щ"] = "o",
  ["з"] = "p",
  ["х"] = "[",
  ["ъ"] = "]",
  ["ф"] = "a",
  ["ы"] = "s",
  ["в"] = "d",
  ["а"] = "f",
  ["п"] = "g",
  ["р"] = "h",
  ["о"] = "j",
  ["л"] = "k",
  ["д"] = "l",
  ["ж"] = ";",
  ["э"] = "'",
  ["я"] = "z",
  ["ч"] = "x",
  ["с"] = "c",
  ["м"] = "v",
  ["и"] = "b",
  ["т"] = "n",
  ["ь"] = "m",
  ["б"] = ",",
  ["ю"] = ".",
}

-- Merge the Russian to Latin keybindings with the general mappings
for russian_key, latin_key in pairs(russian_to_latin) do
  M.general.n[russian_key] = { latin_key, "Russian to Latin keybinding for '" .. russian_key .. "'", opts = { noremap = true } }
end

return M
```

</details>

However when I click russian "н" twice it doesn't transfer into the nvim command "yy", how do I get that to work?

**Comment by siduck** [comment]
no idea, if it works outside of nvchad then i'll look into this...

**Reply by xxfogs** [reply]
Thanks!

**Reply by xxfogs** [reply]
Any success?

**Reply by siduck** [reply]
@xxfogs i didnt try yet, you could ask for help on our discord server, some one could help you
